### ISSUE TITLE
Add support for comma separated lists to the Geo:Postalcode delivery rule

### ISSUE THREAD

**Issue by mbeccati** [revive-adserver/revive-adserver#1416]
The delivery rule currently supports only exact match to a single zip code.

**Comment by erikgeurts** [comment]
Targeting a list of zip codes turns out to be possible by using a regular expression as well, but instead of a comma separated list, the zip codes must be separated with a | characters.

**Comment by mbeccati** [comment]
@erikgeurts Right. Perhaps, it is still a good idea to add the feature as regular expressions might be daunting for the average user.

**Comment by erikgeurts** [comment]
> @erikgeurts Right. Perhaps, it is still a good idea to add the feature as regular expressions might be daunting for the average user.

I agree.

**Comment by mbeccati** [comment]
It is actually not possible to add comma separated values support as it overlaps with the "contains" operator that we already have. I guess it's just a matter of documentation here.
